### ISSUE TITLE
[PS2] Compile Fail at missing gsKit.h

### ISSUE THREAD

**Issue by MasterNama17083** [fgsfdsfgs/sm64-port#104]
I was building PS2 also I've installed PS2DEV toolchain via WSL and then I copied `ps2dev` into the tools folder. 
btw, I've been compiling some platforms and later soon.

How do I fix this?
![image](https://github.com/user-attachments/assets/719faefa-0f9f-4dab-ba99-930b3253c466)

**Comment by fgsfdsfgs** [comment]
~~You seem to be missing gsKit. It used to be mentioned in the Readme that you need to install it separately, but that was removed in a recent PR (#102). I thought it was now bundled with ps2sdk, but apparently not?
Either that or the header file is now called `gskit.h` instead of `gsKit.h` or something.~~
EDIT: Ah, it's something similar. In the Makefile there are two lines that contain `$(PS2DEV)/gskit/`. Change it to `$(PS2DEV)/gsKit/`. I'll commit a change for this as well.

**Comment by MasterNama17083** [comment]
Yeah it seems to be uppercase **K**it letter with this Makefile lol.
And, Again I tried with the reinstalled source code when compiled that says same error.

**Comment by binavik** [comment]
Instructions in the readme say to have the PS2DEV and PS2SDK system variables defined but the code below it to download the prebuilt ps2dev binaries only define PS2SDK. run "export PS2DEV=$(pwd)/ps2dev" in terminal (with the ps2dev folder in the same directory of course) and try compiling it again.

**Comment by LinkToTheAsh** [comment]
@binavik thank you! definining "PS2DEV" `export PS2DEV=$(pwd)/ps2dev` before running make worked for me!

**Comment by godreborn** [comment]
found out from a friend that this apparently no longer works with docker (same error).  I'm not sure if docker uses its own Makefile or the one located in the repo folder, but it was tested in windows and linux.  compiling normally works fine, so I'm not sure.
